app/models/user.rb

```diff
@@ -1,10 +1,15 @@
 class User < ApplicationRecord
   include MetadataHelpers
-
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :invitable, :registerable, :database_authenticatable,
          :recoverable, :rememberable, :trackable, :validatable, :async
 
   belongs_to :account
+
+  validates_inclusion_of :locale, in: %w[en km]
+
+  def locale_en?
```

Comment:
After you implement the drop down menu and remove the conditional you can remove this method